frontend/packages/dev-console/src/utils/__tests__/pipeline-filter-reducer.spec.ts

```diff
@@ -26,43 +26,65 @@ const mockPipelineRuns = [
     },
   },
   { status: { conditions: [{ status: 'Unknown', type: 'Succeeded' }] } },
+  { status: { conditions: [{ reason: 'PipelineRunCancelled' }] } },
+  { status: { conditions: [{ reason: 'TaskRunCancelled' }] } },
 ];
 
-describe('Check PipelineRun Filter Reducer applied to the following:', () => {
+describe('Check PipelineRun Status | Filter Reducer applied to the following:', () => {
   it('1. Pipelinerun with empty status object', () => {
+    const reducerOutput = pipelineRunStatus(mockPipelineRuns[0]);
+    expect(reducerOutput).toBeNull();
+  });
+  it('2. Pipelinerun with empty status object', () => {
     const reducerOutput = pipelineRunFilterReducer(mockPipelineRuns[0]);
     expect(reducerOutput).toBe('-');
   });
-  it('2. Pipelinerun with empty conditions array', () => {
+  it('3. Pipelinerun with empty status object', () => {
```

Comment:
We really should avoid numbering the tests like this imo. This is just painful for maintenance and will look really off if someone failed to properly update all the numbers.

Since you have to make edits to your work for my other comments, could you clean up these and remove the numbers from each of these tests?

Reply:
Removed numbering